src/erlfmt.erl

```diff
@@ -188,6 +188,28 @@ remove_pragma_comment_block([Head | Tail]) ->
         _ -> Tail
     end.
 
+replace_pragma_node(Node0) ->
+    {PreComments0, _, PostComments0} = erlfmt_format:comments_with_pre_dot(Node0),
+    PreComments = replace_pragma_comment_blocks(PreComments0),
+    PostComments = replace_pragma_comment_blocks(PostComments0),
+    Node = erlfmt_scan:put_anno(pre_comments, PreComments, Node0),
+    erlfmt_scan:put_anno(post_comments, PostComments, Node).
+
+replace_pragma_comment_blocks([]) ->
+    [];
+replace_pragma_comment_blocks([{comment, Loc, Comments} | Rest]) ->
+    case replace_pragma_comment_block(Comments) of
+        [] -> replace_pragma_comment_block(Rest);
+        CleanComments -> [{comment, Loc, CleanComments} | replace_pragma_comment_block(Rest)]
+    end.
+
+replace_pragma_comment_block([]) ->
+    [];
+replace_pragma_comment_block(["%% @format" | Tail]) ->
+    ["%%% % @format" | Tail];
+replace_pragma_comment_block([Head | Tail]) ->
+    [Head | replace_pragma_comment_block(Tail)].
```

Comment:
I'm not convinced about forcing the triple comment. What if the surrounding comment uses only double? It would be strange to end up with something like:

```erl
%% Some licence info
%%
%%% % @format
%%
%% @doc ....
```

Reply:
Yes we could replace it with the number of percentages that was there.

Reply:
I have also address this with a default of 3, but if some other number is detected we prefer that.